doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+- Installed Clear Linux OS onto both hard disks on KBL NUC.
+
+.. note::
+      Follow `the installation guide
+      <https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html>`_ to install
+      a Clear Linux OS.
+
+.. _hardware_setup:
+
+Hardware Setup
+==============
+
+.. table:: Hardware Setup
+   :widths: auto
+   :name: Hardware Setup
+
+   +----------------------+------------------+----------------------+-----------------------------------------------------------+
+   | Platform (Intel x86) | Product/kit name | Hardware             | Descriptions                                              |
+   +======================+==================+======================+===========================================================+
+   | Kaby Lake            | NUC7i7DNH        | Processor            | - Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz                |
+   |                      |                  +----------------------+-----------------------------------------------------------+
+   |                      |                  | Graphics             | - UHD Graphics 620                                        |
+   |                      |                  |                      | - Two HDMI* 2.0a po rts supporting 4K at 60 Hz            |
+   |                      |                  +----------------------+-----------------------------------------------------------+
+   |                      |                  | System memory        | - 8GiB SODIMM DDR4 2400 MHz                               |
+   |                      |                  +----------------------+-----------------------------------------------------------+
+   |                      |                  | Storage capabilities | - SATA: 1TB WDC WD10SPZX-22Z                              |
+   |                      |                  |                      | - NVMe: 256G Intel Corporation SSD Pro 7600p/760p/E 6100p |
+   +----------------------+------------------+----------------------+-----------------------------------------------------------+
+
+Software Setup
+==============
+
+.. table:: Software Setup
+   :widths: auto
+   :name: Software Setup
+
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+   | Clear Linux Version | ACRN tag (hypervisor)  | Description                              | Download Link                                                                                  |
+   +=====================+========================+==========================================+================================================================================================+
+   | 29750               |                        | **Host machine:**                        | `link <https://download.clearlinux.org/releases/29750/clear/clear-29750-live-server.iso.xz>`__ |
```

Comment:
Any specific reason why we're using an older version of Clear Linux here?

Reply:
It's just our environment but nothing special.
I've changed this line as below to tell user it's able to use newer version as well:
``A Clear Linux build machine (Version: 29750 or newer) to setup build environment.``